### PR TITLE
Add Commander Genius (Commander Keen Engine)

### PR DESCRIPTION
**Pull request by HerbFargus**

### CODE REVIEW

File: scriptmodules/ports/cgenius.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+function depends_cgenius() {
+    getDepends build-essential libvorbis-dev libogg-dev libsdl2-dev libsdl2-image-dev libgl1-mesa-dev libboost1.55-dev libboost-dev 
+}
+
+function sources_cgenius() {
+    wget -O- -q "https://github.com/gerstrong/Commander-Genius/archive/v180release.tar.gz" | tar -xvz --strip-components=1 -C "$md_build"
+}
+
+function build_cgenius() {
+    cd $md_build
```
Comment (joolswills) [inline scriptmodules/ports/cgenius.sh]: you dont need to cd to $md_build as thats the default folder for the build functions

File: scriptmodules/ports/cgenius.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    cmake -DUSE_SDL2=yes -DCMAKE_INSTALL_PREFIX="$md_inst"
```
Comment (joolswills) [inline scriptmodules/ports/cgenius.sh]: if you are not using "make install" later you probably dont need the install prefix parameter

**Comment by joolswills** [comment]
The above issues are not that important, but I thought to mention them. Will merge.

**Comment by HerbFargus** [comment]
appreciate the feeback

**Pull request merged**